src/platform/tls_schannel.c

```diff
@@ -2495,7 +2531,7 @@ CxPlatTlsParamGet(
     _In_ CXPLAT_TLS* TlsContext,
     _In_ uint32_t Param,
     _Inout_ uint32_t* BufferLength,
-    _Inout_updates_bytes_opt_(*BufferLength)
+    _Out_writes_bytes_opt_(*BufferLength)
```

Comment:
In the case of `QUIC_PARAM_TLS_SCHANNEL_CONTEXT_ATTRIBUTE_W`, `_Inout_updates_bytes_opt_` is correct. It both reads and writes (i.e. updated) the data stored there.

Reply:
So we should update QUIC_GET_PARAM_FN to be _Inout_updates_bytes_opt_. The SAL annotation should match all the way down.

Reply:
Leave that to a separate PR and just undo this change for now?

Reply:
Ok made #1242  to track it.